packages/configurationmap/tests/__snapshots__/HashMap.test.tsx.snap

```diff
@@ -0,0 +1,141 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`HashMap renders array value 1`] = `
+<dl
+  class="css-p4ue6x"
+>
+  <div
+    class="css-9q3tt4"
+  >
+    <dt
+      class="css-1ououz5"
+    >
+      foo
+    </dt>
+    <dd
+      class="css-1pld8at"
+    >
+      foo, bar, baz
+    </dd>
+  </div>
+</dl>
+`;
+
+exports[`HashMap renders boolean value 1`] = `
+<dl
+  class="css-p4ue6x"
+>
+  <div
+    class="css-9q3tt4"
+  >
+    <dt
+      class="css-1ououz5"
+    >
+      foo
+    </dt>
+    <dd
+      class="css-1pld8at"
+    >
+      true
+    </dd>
+  </div>
+</dl>
+`;
+
+exports[`HashMap renders default 1`] = `
+<dl
+  class="css-p4ue6x"
+>
+  <div
+    class="css-9q3tt4"
+  >
+    <dt
+      class="css-1ououz5"
+    >
+      foo
+    </dt>
+    <dd
+      class="css-1pld8at"
+    >
+      bar
+    </dd>
+  </div>
+</dl>
+`;
+
+exports[`HashMap renders empty value 1`] = `
+<dl
+  class="css-p4ue6x"
+>
+  <div
+    class="css-9q3tt4"
+  >
+    <dt
+      class="css-1ououz5"
+    >
+      foo
+    </dt>
+    <dd
+      class="css-1pld8at"
+    >
+      empty
+    </dd>
+  </div>
+</dl>
+`;
+
+exports[`HashMap renders with a headline 1`] = `
+<dl
+  class="css-p4ue6x"
+>
+  <h1
+    class="css-1orn387"
+  >
+    baz
+  </h1>
+  <div
+    class="css-9q3tt4"
+  >
+    <dt
+      class="css-1ououz5"
+    >
+      foo
+    </dt>
+    <dd
+      class="css-1pld8at"
+    >
+      bar
+    </dd>
+  </div>
+</dl>
+`;
+
+exports[`HashMap renders with nested object hash 1`] = `
+<dl
```

Comment:
is that markup here the way we want it?

the double-`dl` makes me suspicious :)
```
dl 
  dl 
    h2 foo
    div
      dt bar
      dd baz
```

Reply:
I agree, this is a bit strange, but I can't find anything in the spec that says it's not ok: https://www.w3.org/TR/html401/struct/lists.html#h-10.3.

I'm open to suggestions for how we might change the parent `dl` though if you have any ideas :)

Reply:
if it's okay for you, it's okay for me! :)